lib/Doctrine/ORM/Query/ResultSetMappingBuilder.php

```diff
@@ -181,6 +181,9 @@ protected function addAllClassFields($class, $alias, $columnAliasMap = [])
         }
     }
 
+    /**
+     * @return bool
+     */
```

Comment:
This method is `private`, you can add a type declaration instead.

Reply:
done